decisions/wc_offmap_decisions.txt

```diff
@@ -625,4 +625,121 @@ offmap_decisions = {
 			}
 		}
 	}
+	wake_old_god_offmap = {
+		only_playable = yes
+		button_name = offmap_boons
+
+		ai_check_interval = 36
+
+		potential = {
+			OR = {
+				AND = {
+					is_offmap_tag = offmap_cthun
+					NOT = { offmap_cthun = { has_status = old_god_dead } }
+				}
+				AND = {
+					is_offmap_tag = offmap_nzoth
+					NOT = { offmap_nzoth = { has_status = old_god_dead } }
+				}
+				AND = {
+					is_offmap_tag = offmap_yoggsaron
+					NOT = { offmap_yoggsaron = { has_status = old_god_dead } }
+				}
+			}
+		}
+		from_potential = {
+			mercenary = no
+			holy_order = no
+			is_landed = yes
+		}
+		allow = {
+			trigger_if = {
+				limit = { is_offmap_tag = offmap_cthun }
+				FROM = {
+					has_offmap_currency = { offmap = offmap_cthun value = 8000 }
+					cthun_offmap_decisions_enabled_trigger = yes
+				}
+				OR = {
+					b_ahnqiraj = { holder_scope = { character = FROM } }
+					offmap_cthun = { has_status = old_god_imprisoned }
+				}
+				offmap_cthun = { 
+					NOR = {
+						has_status = old_god_free
+						has_status = old_god_full_power 
+					} 
+				}
+			}
+			trigger_else_if = {
+				limit = { is_offmap_tag = offmap_nzoth }
+				FROM = {
+					has_offmap_currency = { offmap = offmap_nzoth value = 8000 }
+					nzoth_offmap_decisions_enabled_trigger = yes
+				}
+				OR = {
+					b_sagehold = { holder_scope = { character = FROM } }
+					offmap_nzoth = { has_status = old_god_imprisoned }
+				}
+				offmap_nzoth = {
+					NOR = {
+						has_status = old_god_free
+						has_status = old_god_full_power 
+					} 
+				}
+			}
+			trigger_else = {
+				FROM = {
+					has_offmap_currency = { offmap = offmap_yoggsaron value = 8000 }
+					yogg_saron_offmap_decisions_enabled_trigger = yes
+				}
+				OR = {
+					b_ulduar = { holder_scope = { character = FROM } }
```

Comment:
Should check `has_wonder_upgrade` now. **DO NOT FORGET TO ADD A SCRIPTED TRIGGER**! Like `has_cthun_prison_trigger`.